### DISCUSSION TITLE
How many times the data would be buffered before transfered in Fast-DDS?

### DISCUSSION THREAD

**Discussion by wjhtinger** [eProsima/Fast-DDS#3698] [General]
As I know, the data would be buffered after write() at asynchronous mode，right？So, is there any other data buffer at what condictions and in what layer?

**Comment by JesusPoderoso** [comment]
You can check it in the answer of [this discussion](https://github.com/eProsima/Fast-DDS/discussions/3654#discussioncomment-6372721).
Maybe you are interested in contacting support@eprosima.com to get some commercial support on the matter.
